mitol-django-authentication/mitol/authentication/urls.py

```diff
@@ -0,0 +1,10 @@
+"""URL configurations for authentication"""
+
+from django.urls import path
+
+from mitol.authentication.views.saml import saml_metadata
+
+
+urlpatterns = [
+    path("saml/metadata/", saml_metadata, name="saml-metadata"),
```

Comment:
Do we/will we have any projects that use this auth library, but don't use Touchstone auth? If so, we may want to wrap this route in a condition

Reply:
I'll just move it into a `urls.saml` package rather than have a flag